build/misc.tcl

```diff
@@ -312,6 +312,58 @@ respond "Command:" "d"
 respond "*" ":pdump sys3; ts advent\r"
 respond "*" ":kill\r"
 
+# 350-point ADVENT
+respond "*" ":cwd games\r"
+respond "*" ":dec sys:f40\r"
+respond "*" "adv3sr=adv3sr\r"
+respond "*" "adv3sb=adv3sb\r"
+respond "CORE USED" "\032"
+type ":kill\r"
+respond "*" ":dec sys:loader\r"
+respond "*" "adv3sb\r"
+respond "*" "adv3sr\r"
+respond "*" "/go\r"
+respond "EXIT" ":start\r"
+respond "*" "adv3db.1\032"
```

Comment:
Using `make EMULATOR=klh10`, the build times out here for me:

```
*:dec sys:loader
*adv3sb
*adv3sr
*/go


ADV3SR 23K CORE, 759 WORDS FREE 
LOADER USED 24+4K CORE
EXIT
*:start
INITIALISING...
%FRSOPN File was not found
Unit=1 DSK:TEXT.DAT/ACCESS=SEQIN/MODE=ASCII

Enter new file specs. End with an $(ALT)
*adv3db.1
743067)   .CALL 757630 (IOT)   
IT IS NOW  1:14:36 PM EDT, SUNDAY, MAY 6,2018

The last command timed out.
make: *** [Makefile:46: out/klh10/rp0.dsk] Error 1
```

I think that should be `\033` rather than `\032` - it works OK if I build by hand and press ^Z here.